common/adminBlock/EntryAddEfficiency.go

```diff
@@ -10,23 +10,26 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
-// AddEfficiency Entry -------------------------
+// AddEfficiency is an admin block entry which contains a server identity and an efficiency for that server. The efficiency is specified in
+// hundredths of a percent. So 0% would be specified as 0, 52.34% is specified as 5234, and 100% is specified as 10000.
 type AddEfficiency struct {
-	AdminIDType     uint32 `json:"adminidtype"`
-	IdentityChainID interfaces.IHash
-	Efficiency      uint16
+	AdminIDType     uint32           `json:"adminidtype"` // the type of action in this admin block entry: uint32(TYPE_ADD_FACTOID_EFFICIENCY)
+	IdentityChainID interfaces.IHash // the server identity whose efficiency will be updated
+	Efficiency      uint16           // the efficiency this server will run at from 0 to 10000 (0% to 100.00%)
 }
 
 var _ interfaces.IABEntry = (*AddEfficiency)(nil)
 var _ interfaces.BinaryMarshallable = (*AddEfficiency)(nil)
 
+// Init initializes any nil hashes to the zero hash and sets the object type
 func (e *AddEfficiency) Init() {
 	e.AdminIDType = uint32(e.Type())
 	if e.IdentityChainID == nil {
 		e.IdentityChainID = primitives.NewZeroHash()
 	}
 }
 
+// IsSameAs returns true iff the input object is identical to this object
 func (a *AddEfficiency) IsSameAs(b *AddEfficiency) bool {
```

Comment:
Should be (e *AddEfficiency) ?

Reply:
I was lenient on the IsSameAs functions for receiver names, because they have a nice "a compared to b", but I can update all of them if you'd like to be consistent.

Reply:
done